atlp-rwanda/code-ninjas-fe
# #181414717 (Feature): reset password

### What does this PR do?

Users should be able to  reset a password

### Description of Task to be completed?

A user should be presented with a button and or link to reset their passwords 


### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - switch to [ft-reset-password-181414717](https://github.com/atlp-rwanda/code-ninjas-fe/tree/ft-reset-password-181414717)
  - branch
  - Run `yarn install`: to install all dependencies
  - Run `yarn dev`: to start the application
  - Open your browser & visit  `/forgot-password` route.  enter email which has account on barefoot nomad. you will get email(message) with a link to reset password, click it.
  - Now you will be redirected to /reset-password/?token=............(token). you will see two input boxes fill them with your new password then press **reset password** button. 

`

Routes:


`



[/forgot-password](https)
Forgot password PAGE  `All users`

[/reset-passwordl](https)
Reset password PAGE  `All users`

`

### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[#181414717](https://www.pivotaltracker.com/story/show/181414717)
### Screenshots (if appropriate)
### Questions:
